semgrep_output_v1.atd

```diff
@@ -1039,29 +1040,35 @@ type scan_request = {
 }
 
 (* Response by the backend to the CLI to the POST /scans *)
-(* TODO
 type scan_response = {
-    scan: scan_response_bis;
+    scan: scan_info;
+    scan_config: scan_configuration;
+    engine_config: engine_configuration;
 }
-type scan_response_bis = {
-    (* the scan id *)
-    id: string;
-    (* since 1.37.0 *)
-    ?enabled_products: product list;
+
+(* meta info about the scan *)
+type scan_info = {
+    id: string; (* the scan id *)
+    deployment_id: int;
+    deployment_name: string;
+}
+
+(* config specific to the scan, eg *)
+type scan_configuration = {
+    enabled_products: product list;
+    rules: raw_json; (* can we type this better *)
+    ~ignored_files: string list;
+    ~triage_ignored_syntactic_ids: string list;
+    ~triage_ignored_match_based_ids: string list;
 }
-*)
 
-(* TODO? seems used only for dry_run and passed urlencoded to /scans/config?xxx
```

Comment:
removing this since my goal is to remove this endpoint entirely in the next 1-2 versions of the cli

Reply:
Hmm, I think this is used for the 'semgrep scan --config policy'
that is we're not really starting a CI scan (and upload findings), but we want to fetch the rules from the ruleboard

Reply:
Yeah, I think we should update the CLI to use the same endpoint as I'm adding for CI at the moment. But probably in a separate commit.